statsmodels/tsa/statespace/kalman_filter.py

```diff
@@ -1271,3 +1576,149 @@ def _predict(self, nstatic, ndynamic, nforecast, model):
         results = FilterResults(model)
         results.update_filter(kfilter)
         return results
+
+
+class PredictionResults(FilterResults):
```

Comment:
Is this a wrapper class similar to our Results wrappers with the same pattern of `__getattr__`?